external/@wordpress/blocks/index.js

```diff
@@ -8,7 +10,14 @@ export { default as Editable } from './components/editable';
  * @param  {Object} settings  Block settings
  */
 export function registerBlock( namespace, block, settings ) {
+	if ( ! blocks.namespace ) {
+		blocks[ namespace ] = {};
+	}
 
+	blocks[ namespace ][ block ] = Object.assign(
+		{ name: block, namespace },
```

Comment:
If we call it `name` on the object, should we just accept the argument labeled as `name` as well?